spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
```

Comment:
- [  ] Please you should add a test for validations for (user,post and comments model) it should look something like this 
`
describe 'validations' do
  it { should validate_presence_of(:name) }
end
`